cypress/support/commands.js

```diff
@@ -126,6 +146,20 @@ Cypress.Commands.add("assertUserAEurBalanceOnUI", (balance, options = {}) => {
         .should("equal", balance.toString());
 });
 
+// assert user balance on UI.
+Cypress.Commands.add("assertUserEthBalanceOnUI", (_expectedEth, decimals, options = {}) => {
```

Comment:
how about setting the default value for the `decimals` param? `(_expectedEth, decimals=12, options = {})`